<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recv_image</class>
 <widget class="QDialog" name="recv_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SaveAs">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>纠错</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_emptyTemp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>重新接收</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
